t/067-req-socket.t

```diff
@@ -1096,3 +1096,73 @@ done
 --- grep_error_log_out
 lua finalize socket
 GC cycle done
+
+
+
+=== TEST 18: receiveany
+--- config
+    location = /t {
+        content_by_lua_block '
```

Comment:
I am afraid that `content_by_lua_block` uses a different syntax, see https://github.com/openresty/lua-nginx-module/blob/master/t/058-tcp-socket.t#L3689 as an example.

You should use `{}` instead of `''`, and `\r\n` instead of `\\r\\n`.

Reply:
You're right, sorry about that. Fixed.